drivers/src/nrfx_nvmc.c

```diff
@@ -189,27 +189,42 @@ static uint32_t partial_word_create(uint32_t addr, uint8_t const * bytes, uint32
 
 static void nvmc_readonly_mode_set(void)
 {
-#if defined(NRF_TRUSTZONE_NONSECURE)
+    /*
```

Comment:
@anangl this is so confusing because it seems that the comment applies to the #ifdef right below, but it does not!